examples/resources/awscc_applicationautoscaling_scalable_target/applicationautoscaling_scalable_target_keyspaces.tf

```
@@ -0,0 +1,31 @@
+resource "awscc_applicationautoscaling_scalable_target" "example" {
+  max_capacity       = 20
+  min_capacity       = 1
+  resource_id        = "keyspace/example/table/example"
```

```suggestion
  resource_id        = "keyspace/example/table/${awscc_cassandra_table.example.table_name}"
```

Let's refer the relative table name from the existing resource

---

Changes done, re-ran docs. Thank you.